src/install/PackageManager/CommandLineArguments.zig

```diff
@@ -651,9 +651,10 @@ pub fn parse(allocator: std.mem.Allocator, comptime subcommand: Subcommand) !Com
     }
 
     var cli = CommandLineArguments{};
+    cli.positionals = args.positionals();
     cli.yarn = args.flag("--yarn");
     cli.production = args.flag("--production");
-    cli.frozen_lockfile = args.flag("--frozen-lockfile");
+    cli.frozen_lockfile = args.flag("--frozen-lockfile") or strings.eqlComptime(cli.positionals[0], "ci");
```

Comment:
Does this mean if you type `bun install ci` it would become `--frozen-lockfile`?